crates/re_space_view/src/view_property_ui.rs

```diff
@@ -164,38 +164,35 @@ fn view_property_component_ui(
             .show_hierarchical(ui, singleline_list_item_content)
     };
 
-    list_item_response = list_item_response.on_hover_ui(|ui| {
+    list_item_response.on_hover_ui(|ui| {
         let id = egui::Id::new((archetype_name, field.display_name));
         ui.markdown_ui(id, field.docstring_md);
     });
-
-    view_property_context_menu(
-        ctx.viewer_ctx,
-        &list_item_response,
-        blueprint_path,
-        component_name,
-        component_results,
-    );
 }
 
-fn view_property_context_menu(
+fn menu_more(
     ctx: &ViewerContext<'_>,
-    list_item_response: &egui::Response,
+    ui: &mut egui::Ui,
     blueprint_path: &re_log_types::EntityPath,
     component_name: ComponentName,
     component_results: &re_query::LatestAtComponentResults,
 ) {
-    list_item_response.context_menu(|ui| {
-        if ui.button("Reset to default blueprint.")
+    let resolver = ctx.blueprint_db().resolver();
+
+    let property_differs_from_default = component_results.raw(resolver, component_name)
+        != ctx.raw_latest_at_in_default_blueprint(blueprint_path, component_name);
+    ui.add_enabled_ui(property_differs_from_default, |ui| {
+    if ui.button("Reset to default blueprint.")
```

Comment:
No trailing "." in menu items!

```suggestion
    if ui.button("Reset to default blueprint")
```